kashgari/tasks/labeling/base_model.py

```diff
@@ -92,8 +92,15 @@ def evaluate(self,
         Returns:
 
         """
-        y_pred = self.predict(x_data, batch_size=batch_size)
-        y_true = [seq[:len(y_pred[index])] for index, seq in enumerate(y_data)]
+        temp_y_pred = self.predict(x_data, batch_size=batch_size)
+        temp_y_true = [seq[:len(y_pred[index])] for index, seq in enumerate(y_data)]
```

Comment:
I think this line has some errors. You defined y_pred after this line. But you used it before defining it. This will cause an undefined variable error. Please checkout the lint report. 
- [Github build log](https://github.com/BrikerMan/Kashgari/pull/232/checks?check_run_id=213813551)
- [Travis build log](https://travis-ci.com/BrikerMan/Kashgari/jobs/231671385)

Reply:
yes, you are right, I will correct this